atomic_physics/tests/test_spin_half.py

```diff
@@ -0,0 +1,41 @@
+"""Test Spin 1/2 nuclei"""
+import unittest
+from atomic_physics.ions import ba133
+from atomic_physics.utils import Lande_g
+import numpy as np
+import scipy.constants as ct
+
+
+def _gF(F, J, I, gJ):
+    return gJ * (F * (F + 1) + J * (J + 1) - I * (I + 1)) / (2 * F * (F + 1))
+
+
+class TestSpinHalf(unittest.TestCase):
```

Comment:
can we use `pytest` here (see how it's done in `ionics_fits`)

Reply:
(and make these functions rather than classes)

Reply:
Yes - though isn't it best to stay consistent with the current tests in this package? Or is there a plan to change those as well?

Reply:
I'd forgotten we had tests. In that case, yes, stick with this style. Thanks for the correction!

Reply:
Ah - cool! May I merge this now?